### ISSUE TITLE
Allow slower playback other than 0.5 or 1.0

### ISSUE THREAD

**Issue by evandrocoan** [blmage/duolingo-tts-controls#122]
The default slower audio only allows 1.0, but it is not enough because the default slow speed is still too fast for some languages.

![image](https://github.com/user-attachments/assets/fdbfb5ac-3a4c-48fb-ab74-727c0d2c70e0) ![image](https://github.com/user-attachments/assets/25f8ce91-0ceb-457e-a767-f1df4b34a53d)

It would be a line to allow slower speeds other than 0.5 and 1.0, such as 0.1 and 0.5.

I hope to change this line from 1.0 : 0.5 to 0.5 : 0.1 will be enough:
https://github.com/blmage/duolingo-tts-controls/blob/0cabed3db0425c3eeaa84d63ce884cb4aca027ac/src/tts.js#L40-L43

I could not test it because I could not build the extension:
```
npm install
npm ERR! code E401
npm ERR! Incorrect or missing password.
npm ERR! If you were trying to login, change your password, create an
npm ERR! authentication token or enable two-factor authentication then
npm ERR! that means you likely typed your password in incorrectly.
npm ERR! Please try again, or recover your password at:
npm ERR!     https://www.npmjs.com/forgot
npm ERR!
npm ERR! If you were doing some other operation then your saved credentials are
npm ERR! probably out of date. To correct this please try logging in again with:
npm ERR!     npm login
```

**Comment by evandrocoan** [comment]
I managed to edit the extension minified code, changing `Yo=e=>Za(Ua,Da===e?1:.5)`  to `Yo=e=>.1` inside `ui.js`. However, the user interface only allows up to 0.5 as the minimum. But 0.5 works fine for the slower audio, as the original audio is already slower.

**Comment by blmage** [comment]
Thanks for the suggestion! This should be available in version **2.7.0** soon.

> I could not test it because I could not build the extension:

This is because the extension uses FontAwesome Pro icons, and installing the corresponding package requires a private token (if needed at some point, I may consider solutions to work around this limitation).

> However, the user interface only allows up to 0.5 as the minimum.

This is a hard limit from the [Howler.js](https://github.com/goldfire/howler.js#documentation) library used by Duolingo, which [does not allow rate values below 0.5](https://github.com/goldfire/howler.js?tab=readme-ov-file#rate-number-10).

**Comment by evandrocoan** [comment]
Thanks for the fast response and for adding it to the stable version. I opened a feature request on Howler.js to allow values below 0.5 to the slider. Let's wait and see what they think about it.
